### ISSUE TITLE
Demo server is down

### ISSUE THREAD

**Issue by nagygergo** [brmson/yodaqa#73]
Hi! 

As pointed out in #64 the demo app is down probably due to the outage of the DBpedia instance. I hope you can resolve it as soon as you can. :)

**Comment by bjascob** [comment]
Trying to run YodaQA from my PC today for the fist time in a long while and I'm seeing...
http://yodaqa.felk.cvut.cz/dbp-labels1 or http://yodaqa.felk.cvut.cz/dbp-labels2 label lookup query (temporarily?) failed, retrying in a moment...
and
Server at http://yodaqa.felk.cvut.cz/apache/solr returned non ok status:503, message:Service Temporarily Unavailable

Are the remote services on http://yodaqa.felk.cvut.cz still planned to be generally available and just down temporarily or have they been taken off-line?  If there is an alternate address to use please let me know.

**Comment by k0105** [comment]
Should be back up. For me it works (even though it seems performance has slightly deteriorated). Can we close this?

**Comment by bjascob** [comment]
Yes.  Thanks.  Looks like everything is up and running so this can be closed.

**Comment by k0105** [comment]
Perfect. Thank you.

**Comment by k0105** [comment]
Seems like the backend broke again, cmp. https://github.com/brmson/yodaqa/issues/74

**Comment by limm33** [comment]
Demo server is down，can not be used properly

**Comment by kickbox** [comment]
Still down.

**Comment by chc200** [comment]
still down

**Comment by murphp15** [comment]
It seems to be down again.
